test/resources/privatekey.pem

```diff
@@ -0,0 +1 @@
+PRIVATE KEY FILE
```

Comment:
looks like for some reason windows adds an extra `\r` somewhere in the private key file and that's why the tests are failing on windows: https://travis-ci.com/github/adobe/asset-compute-client/jobs/353755542#L139

Reply:
blarg

Reply:
(Trivia of the day: Windows created and Linux created files have different line endings. `\r` corresponds to CR, and CR LF is the new line on Windows machines (aka \r\n). Historically, this comes from commands send to printers when printers/tty started to exist.
That is why there is a setting somewhere in git to choose which line ending to use, for instance.)